chapters/hardware-software-interface/x86-architecture-family/drills/tasks/min/README.md

```diff
@@ -0,0 +1,7 @@
+# BONUS 1: Min
```

Comment:
```suggestion
# Min
```
The fact that it's a bonus task is something related to UPB, that will be specified in the `config.yaml`.